artemis-server/src/main/java/org/apache/activemq/artemis/core/deployers/impl/FileConfigurationParser.java

```diff
@@ -923,7 +923,6 @@ private Map<String, String> getMapOfChildPropertyElements(Node item) {
       }
       return properties;
    }
-
```

Comment:
I guess this line was not removed on purpose

Reply:
indeed, good catch, thanks.